bambi/backend/pymc.py

```diff
@@ -25,6 +25,7 @@ class PyMC3Model:
         "log": tt.exp,
         "logit": logit,
         "probit": probit,
+        "tan_2": lambda x: 2 * tt.arctan(x),
```

Comment:
Can we make this a regular function? We're still not using any pickling but this can be an impediment. See #400.